package.json

```diff
@@ -6,6 +6,14 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/classnames": "^2.2.10",
+    "@types/jest": "^26.0.14",
+    "@types/node": "^14.11.1",
+    "@types/prop-types": "^15.7.3",
+    "@types/react": "^16.9.49",
+    "@types/react-dom": "^16.9.8",
+    "@types/react-router-dom": "^5.1.6",
```

Comment:
There seems to be an issue with the merge with main. Please remove these `@types` from the dependencies and keep them in your `devDependencies`. I rebased main with your branch locally it fixed the issue! Whenever you're fixing merge conflicts, please be aware of what the changes are.